src/App.js

```diff
@@ -23,6 +25,14 @@ function App() {
             <Route path="/" element={<Home />}></Route>
             <Route path="/topics" element={<Topics />}></Route>
             <Route path="/topics/:topic" element={<Home />}></Route>
+            <Route
+              path="/articles/:article_Id"
+              element={<SingleArticle />}
+            ></Route>
+            <Route
```

Comment:
Do you need this route? There isn't a time when you 're going to render just comments, so I think it would be ok to pass down the article_id in props rather than use useparams. It works as it is, just a thought :)